src/Chapter1.hs

```diff
@@ -620,8 +633,10 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = div last2 10 + mod last2 10
+    where last2 = (abs n) `mod` 100
```

Comment:
Also, you can remove `()`, they are redundant in this case :slightly_smiling_face: 
```suggestion
    where last2 = abs n `mod` 100
```